.github/PULL_REQUEST_TEMPLATE/pull_request_bugfix.md

```diff
@@ -0,0 +1,31 @@
+Thank you for your proposed bug fix and for contributing to Quire! Please complete the form below to submit your pull request for review. 
+
+### Checklist 
+
+Put an `x` next to the boxes that apply.
+
+- [ ] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file
+
+- [ ] I have made my changes in a new branch and not directly in the main branch
+
+- [ ] I have added tests that prove my fix is effective
```

Comment:
Did we talk about this? Do we want to mention testing when the team hasn't really defined how it works now or how someone would go about adding tests like this?

Reply:
@geealbers I'll just make a note to add to the pull request templates once we have those details finalized.